.github/workflows/deploy.yml

```
@@ -0,0 +1,43 @@
+        uses: docker/login-action@v3
+        with:
+          username: ${{secrets.DOCKERHUB_USERNAME}}
+          password: ${{secrets.DOCKERHUB_TOKEN}}
+
+      # Docker image 빌드 및 DockerHub에 push
+      - name: Build docker image and push it to Docker Hub
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: hyun7586/moamoa-server
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)